src/ContactsTabManager.php

```diff
@@ -156,7 +141,7 @@ public function getTabs(UserInterface $contact) {
   /**
    * {@inheritdoc}
    */
-  public function getBlocks(ContactTabInterface $tab, UserInterface $contact = NULL, $verify = TRUE) {
+  public function getBlocks(ContactTabInterface $tab, $apply_context = FALSE, UserInterface $contact = NULL) {
     $blocks = $tab->getBlockPlugins();
     if (empty($blocks)) {
```

Comment:
I think we need to differentiate between empty (i.e. no valid blocks) and not attempted... Otherwise a call to `getTab()` followed by `getBlocks()` could result in the whole process happening twice to return empty.